packages/ui5-info/test/testdata/official-response-latest-out-of-maintenance.json

```diff
@@ -0,0 +1,31 @@
+{
+    "latest": {
+        "version": "1.125.1",
+        "support": "Maintenance",
```

Comment:
Shouldnt this be `Out of Maintenance` i.e. 1.126 to test the specific case?

Reply:
Nope, I'm testing that 1.126 is the higher version but out of maintenance, i.e replicating incident from last week.

Reply:
OK but what will happen if "latest" is `out of maintenance` ?

Reply:
That shouldn't happen normally right?

Reply:
Test added in [03a5023](https://github.com/SAP/open-ux-tools/pull/2166/commits/03a5023ee5d196a664b9f599e5c49ae8a9dc8314).